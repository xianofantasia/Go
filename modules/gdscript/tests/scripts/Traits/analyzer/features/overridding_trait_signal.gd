trait SomeTrait:
	signal some_signal(to_emit: Node)

class SomeClass:
	uses SomeTrait
	signal some_signal(to_emit: Object)

func test():
	print("ok")
