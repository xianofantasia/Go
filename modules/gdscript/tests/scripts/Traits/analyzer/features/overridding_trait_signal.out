GDTEST_OK
~~ WARNING at line 6: (UNUSED_SIGNAL) The signal "some_signal" is declared but never explicitly used in the class.
ok
